base/src/org/compiere/model/MOrder.java

```diff
@@ -2243,6 +2249,12 @@ public String reopenIt() {
 			return "Failed to update reservations";
 		}
 
+		if (!calculateTaxTotal())
+		{
+			m_processMsg = "Error calculating tax";
```

Comment:
@nsarlabos Could you create messages in the dictionary so that the response to the user is multi-language?

m_processMsg = "@ErrorCalculatingTax@";